src/coreclr/scripts/genEventing.py

```diff
@@ -80,6 +80,38 @@ def __init__(self, runtime):
     "BYTE"              : "BYTE",
 }
 
+coreCLRUserEventDataTypeMapping={
```

Comment:
```suggestion
coreclrUserEventDataTypeMapping={
```

I would recase as "coreclr", one word.

Reply:
I generally agree, but there are a few other variables named this way so I am following the convention in this file